PWGJE/Tasks/jetfinderQA.cxx

```diff
@@ -435,16 +437,36 @@ struct JetFinderQATask {
       return;
     }
     registry.fill(HIST("h_collisions"), 1.5);
+    fillTrackHistograms(tracks);
+  }
+  PROCESS_SWITCH(JetFinderQATask, processTracks, "QA for charged tracks", false);
+
+  void processTracksWeighted(soa::Join<aod::JCollisions, aod::JMcCollisionLbs>::iterator const& collision,
+                             aod::JMcCollisions const& mcCollisions,
+                             soa::Filtered<JetTracks> const& tracks)
+  {
+    registry.fill(HIST("h_collisions"), 0.5);
+    registry.fill(HIST("h_collisions_weighted"), 0.5, collision.mcCollision().weight());
```

Comment:
out of interest what is the benefit of this table?

Reply:
So that we can show the weighted spectra correctly normalised by the number of events (which needs to be filled weighted as well in this case). This sort of normalisation I think will be the best to compare to the MB MC spectra.
I thought that it would still be useful to fill h_collisions unweighted if the option doprocessTracksWeighted is enabled, so I keep this histogram.